templates/experiments/filters.html

```diff
@@ -129,7 +133,8 @@
           'last_message': {type: 'timestamp', operators: fieldTypeFilters.timestamp, label: 'Last Message'},
           'tags': {type: 'choice', operators: fieldTypeFilters.choice, options: tags, label: 'Tags' },
           'versions': {type: 'choice', operators: fieldTypeFilters.choice.filter(op => op === 'any of' || op === 'excludes'), options: versionsList, label: 'Versions'},
-          'channels': {type: 'choice', operators: fieldTypeFilters.choice, options: channelsList, label: 'Channels'}
+          'channels': {type: 'choice', operators: fieldTypeFilters.choice, options: channelsList, label: 'Channels'},
+          'date_range': {type: 'datetime', operators: fieldTypeFilters.datetime, label: 'Date Range'}
```

Comment:
I'm wondering if it's strictly necessary to have a `date_range` filter. Could we not use the `last_message` filter for selected date range, since it ultimately applies to the last message. If there are separate filters that translates to the same column, there might exist a scenario where I can do something like

```
last_message before today && date_range after today
```

which doesn't really make sense. Happy to hear your thoughts